webpack.config.js

```diff
@@ -0,0 +1,191 @@
+            chunkLoadTimeout: 15000,
+            clean: true,
+        },
+        cache: {
+            type: 'filesystem',
+            cacheLocation: path.resolve(__dirname, `.yarn/webpack/${argv.mode}`),
+            buildDependencies: {
+                config: [__filename],
+            },
+        },
+
+        optimization: {
+            emitOnErrors: false,
+            minimizer: [
+                new TerserPlugin({
+                    extractComments: false,
+                    terserOptions: {
+                        output: {
+                            ascii_only: true,
+                            comments: false,
+                        },
+                    },
+                }),
+            ],
+            splitChunks: {
+                chunks: "all",
+                cacheGroups: {
+                    packages: {
+                        test: /[\\/]node_modules[\\/]/,
+                        idHint: 'packages',
+                    },
+                    api: {
+                        priority: 1,
+                        test: /[\\/]ApiClient[\\/]/,
+                        idHint: 'api',
+                        enforce: true,
+                    },
+                    styles: {
+                        test: /\.(scss)$/,
+                        idHint: 'styles',
+                        enforce: true,
+                    },
+                },
+            },
+        },
+
+        devServer: {
+            static: path.join(__dirname, "dist"),
+            compress: true,
+            hot: true,
+            host: "0.0.0.0", // technically insecure? don't put nudes in your app, helps to test with mobile
+            port: 8080,
+            historyApiFallback: true,
+            stats: createStats(false),
+        },
+
+        stats: createStats(true),
+
+        plugins: [
+            new CopyPlugin({
+                patterns: [
+                    {
+                        from: "public",
+                        toType: "dir",
+                        globOptions: {
+                            ignore: (prodLike
+                                ? ["**/channel.json"]
+                                : [])
+                        },
+                    },
+                ],
+            }),
+            new ForkTsCheckerWebpackPlugin(),
+            new DefinePlugin({
+                'API_VERSION': JSON.stringify(
+                    require("./src/ApiClient/generatedcode/swagger.json").info.version
+                ),
+                'VERSION': JSON.stringify(
+                    github ? env.GITHUB_SHA : require("./package.json").version
+                ),
+                'MODE': JSON.stringify(prodLike ? (github ? "GITHUB" : "PROD") : "DEV"),
+                // The basepath remains /app because its for the router which is located at /app/
+                'DEFAULT_BASEPATH': JSON.stringify(github ? "/app/" : (prodLike ? "/app/" : "/")),
```

Comment:
```suggestion
                'DEFAULT_BASEPATH': JSON.stringify(prodLike ? "/app/" : "/")),
```

Github is prod, not sure why i did it that way originally